weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooks.ts

```diff
@@ -475,6 +475,7 @@ const useCallsStats = (
   project: string,
   filter: CallFilter,
   query?: Query,
+  limit?: number,
```

Comment:
I think this is kind of a counterintuitive parameter, I think it would be worth adding a comment explaining its purpose to trigger the optimized query, and maybe consider a wrapper method named something like `useCallsExist`.

Reply:
interesting, I agree it's counter intuitive. I can add that hook, but I also forsee adding limit to other stats queries. For example, for a string filter query, most use cases include either "show me some examples of this case" or "i know exactly what i'm looking for", both don't really benefit from the exact "497 matches"; I think we can get away with "100+" for example. This should unlock performance down the road. Happy to add comments around it though.